src/ngraph/op/constant.hpp

```diff
@@ -244,6 +244,17 @@ namespace ngraph
                     set_output_type(0, m_element_type, m_shape);
                 }
 
+                bool visit_attributes(AttributeVisitor& visitor) override;
+
+                bool evaluate(const HostTensorVector& outputs,
+                              const HostTensorVector& inputs) override;
+
+                // Don't constant fold a constant; it would make a copy
+                bool constant_fold(OutputVector& outputs, const OutputVector& inputs) override
+                {
+                    return false;
```

Comment:
Could you explain cases when constant folding can create a copy?

Reply:
It's already a constant. If we were to "fold it" it would be making a copy of the constant and replacing it in the graph, which doesn't make any sense.